content/en/blog/_posts/2024-05-13-Gateway-API-v1.1/index.md

```diff
@@ -0,0 +1,199 @@
+- TLS is changed to `Validation`, and the type name changes from
+BackendTLSPolicyConfig to  `BackendTLSPolicyValidation`.
+- CACertRefs becomes `CACertificateRefs` in what is now `BackendTLSPolicyValidation`
+- WellKnownCACerts becomes `WellKnownCACertificates` and the type name
+changes from WellKnownCACertType to `WellKnownCACertificatesType`
+- the constant WellKnownCACertSystem becomes `WellKnownCACertificatesSystem`
+
+Leading Contributors: Candace Holman, Rob Scott
+
+### Everything else
+
+For a full list of the changes included in this release, please refer to the
+[v1.1.0 release
+notes](https://github.com/kubernetes-sigs/gateway-api/releases/tag/v1.1.0).
+
+## Gateway API background
+
+The idea of Gateway API was initially [proposed](https://youtu.be/Ne9UJL6irXY?si=wgtC9w8PMB5ZHil2)
+at the 2019 KubeCon San Diego as the next generation
+of Ingress API. Since then, an incredible community has formed to develop what
+has likely become the
+[most collaborative API in Kubernetes history](https://www.youtube.com/watch?v=V3Vu_FWb4l4).
+Over 200 people have contributed to this API so far, and that number continues
+to grow.
+
+The maintainers
+would like to thank _everyone_ who's contributed to Gateway API, whether in the
+form of commits to the repo, discussion, ideas, or general support. We literally
+couldn't have gotten this far without the support of this dedicated and active
+community.
+
+## Try it out
+
+Unlike other Kubernetes APIs, you don't need to upgrade to the latest version of
+Kubernetes to get the latest version of Gateway API. As long as you're running
+one of the 5 most recent minor versions of Kubernetes (1.25+), you'll be able to
+get up and running with the latest version of Gateway API.
+
+To try out the API, follow our [Getting Started Guide](https://gateway-api.sigs.k8s.io/guides/).
+
+## Get involved
+
+There are lots of opportunities to get involved and help define the future of
+Kubernetes routing APIs for both ingress and service mesh.
+
+* Check out the [user guides](https://gateway-api.sigs.k8s.io/guides) to see what use-cases can be addressed. 
+* Try out one of the [existing Gateway controllers ](https://gateway-api.sigs.k8s.io/implementations/)
+* Or [join us in the community](https://gateway-api.sigs.k8s.io/contributing/)
+and help us build the future of Gateway API together!
+
+## Related Kubernetes Blogs
+* [New Experimental Features in Gateway API v1.0](https://kubernetes.io/blog/2023/11/28/gateway-api-ga/)
+11/2023
+* [Gateway API v1.0: GA Release](https://kubernetes.io/blog/2023/10/31/gateway-api-ga/)
+10/2023
+* [Introducing ingress2gateway; Simplifying Upgrades to Gateway API](https://kubernetes.io/blog/2023/10/25/introducing-ingress2gateway/)
+10/2023
+* [Gateway API v0.8.0: Introducing Service Mesh Support](https://kubernetes.io/blog/2023/08/29/gateway-api-v0-8/)
+08/2022
+* [Kubernetes Gateway API Graduates to Beta](https://kubernetes.io/blog/2022/07/13/gateway-api-graduates-to-beta/)
+07/2022
```

Comment:
```suggestion
```
I wonder if the later release articles are sufficient additional reads here as this one is nearly 2 years old and maybe has old information compared to the others?

Reply:
Agreed.